views/ABViewTextCore.js

```diff
@@ -202,6 +202,9 @@ module.exports = class ABViewTextCore extends ABViewWidget {
          result = result.replace(template, data);
       });
 
+      // Support {uuid} tag in tempalte
+      result = result.replace(/{uuid}/g, rowData.uuid);
```

Comment:
Does it be better ? If you use `rowData[object.PK()]` to support objects that have different PK